### DISCUSSION TITLE
Bug: Default quotes keep getting re-added

### DISCUSSION THREAD

**Discussion by j-freddy** [bluecaret/carettab#356] [Help and Support] [answered]
I am using the quotes widget with 1 quote only. However, these 5 default quotes keep re-appearing. I _believe_ they get automatically re-added every 24 hours.

<img width="647" alt="Screenshot 2023-08-12 at 00 26 15" src="https://github.com/bluecaret/carettab/assets/76404803/7ced0af6-734b-4ca5-8d00-a14029ab8f6a">

**Comment by bluecaret** [comment]
Very timely bug report! The quotes are not correctly being deleted. I have the same issue on Todo lists that I am fixing (literally) right now, and I think this is the same issue. I should have a fix pushed out today, and Chrome takes a couple days to approve usually, keep an eye out. Thank you!

**Comment by bluecaret** [comment, marked as answer]
This will have been resolved as of version 4.1.1. Thanks!
